main.js

```diff
@@ -14,11 +14,24 @@ var status = ['Applying', 'Under Interview', 'Exam Pending', 'Admitted', 'Probat
 async function acceptStudent(id, fullName, age, address){
     let attributes = { ID:id, Name: fullName, Age: age, Address: address };
     await db.put(id, attributes);
-    return await db.get(id, function(err, value){
+    await db.get(id, function(err, value){
         if(err){
             console.log(err);
         } else{
             console.log(value, status[0]);
         }
     })
+    var scheduleDate = new Date('March 1, 2021 08:30:00');
+    return await scheduleInterview(id, scheduleDate);
 }
+
+async function scheduleInterview(id, scheduleDate){
+    return await db.get(id, function(err, value){
+        if(err){
+            console.log(err);
+        } else{
+            console.log(value, status[1]);
+            console.log('Interview date on', scheduleDate);
+        }
+    })
```

Comment:
use await to get the student object.
```suggestion
    var student = await db.get(id);
    student.interviewDate = scheduleDate;
    await db.put(id, student);
    console.log('Interview date on', scheduleDate);
```

Reply:
it is updated sir, but with few changes for the console.log().